src/DDTrace/Integrations/Slim/SlimIntegration.php

```diff
@@ -34,6 +34,7 @@ function ($app) use ($integration, $appName) {
                 // Overwrite root span info
                 $rootSpan = \DDTrace\root_span();
                 $integration->addTraceAnalyticsIfEnabled($rootSpan);
+                $rootSpan->name = 'slim.request';
```

Comment:
Q: Can this be overwritten?

A lot of our notebooks & dashboarding are configured on web.request, so it would be a shame if everything needs to be updated again

Reply:
Tagging @PROFeNoM and @bwoebi for visibility

Or if you require me to create an issue by all means I can do that :D

Reply:
You can easily overwrite it with `\DDTrace\root_span()->name = 'web.request';` at the end of your entrypoint for example.